tests/sea_level_postprocessor/screen-output

```diff
@@ -0,0 +1,37 @@
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+Number of active cells: 96 (on 1 levels)
+Number of degrees of freedom: 9,030 (2,910+150+150+970+970+970+970+970+970)
+
+Number of mesh deformation degrees of freedom: 450
+   Solving mesh displacement system... 0 iterations.
+*** Timestep 0:  t=0 years, dt=0 years
+   Solving mesh displacement system... 0 iterations.
+
+   Postprocessing:
+     Compositions min/max/mass:            0/0/0 // 0/0/0 // 0/0/0 // 0/0/0 // 0/0/0 // 0/0/0
+     Temperature min/avg/max:              293 K, 293 K, 293 K
+     RMS, max velocity:                    0 m/year, 0 m/year
+     Topography min/max:                   -2.794e-09 m, 0 m
+     Density at top/bottom of domain:      3300 kg/m^3, 3300 kg/m^3
+     Writing geoid anomaly:                output-sea_level_postprocessor/geoid_anomaly.00000
```

Comment:
One last request: The postprocessor writes this file to disk, but it isn't part of your test. As a consequence, *what* the postprocessor computes is not actually tested, just that it runs. Could you add the file to `tests/sea_level_postprocessor/` so that this is also tested?